pkg/repository/local_git.go

```diff
@@ -24,6 +29,23 @@ type LocalExec struct{}
 
 func (*LocalExec) RunCommand(ctx context.Context, dir string, name string, arg ...string) ([]byte, error) {
 	c := exec.CommandContext(ctx, name, arg...)
+	c.Cancel = func() error {
+		oktetoLog.Debugf("terminating %s - %s/%s", c.String(), dir, name)
+		if err := c.Process.Signal(syscall.SIGTERM); err != nil {
```

Comment:
Do we know if this is working on windows? We have been facing issues with signals on windows before and leaving zombie processes because of that